3. Installation/3. Docker Containers/README.md

```diff
@@ -10,18 +10,18 @@ For people new to docker here's a quick primer: Docker is a program to allow oth
 This guide is designed for everyone, however, it is intentionally detailed to help new users who may need a little extra guidance. If you're experienced with things like docker, nginx, etc. you may prefer one of the other deployment guides found elsewhere on this wiki.
 
 ### What we'll cover in this guide:
-1. [Securing the server: Firewall basics (optional)](#1-securing-the-server-firewall-basics-optional-recommended)
-2. [Securing the server: Fail2Ban IDS (optional)](#2-securing-the-server-fail2ban-optional-recommended)
+1. [Securing the server: Firewall basics (optional)](#1-securing-the-server-firewall-basics-optional-recommended-)
```

Comment:
The last `-` being added doesn't seem right, what is causing that to be required?